src/unix/timer.c

```diff
@@ -104,15 +104,12 @@ int uv_timer_stop(uv_timer_t* handle) {
 
 
 int uv_timer_again(uv_timer_t* handle) {
-  if (handle->timer_cb == NULL)
+  /* If timer_cb is NULL that means that the timer was never started. */
+  if (handle->timer_cb == NULL || handle->repeat == 0)
```

Comment:
If we're adding a comment about timer_cb maybe it's a good idea also comment the repeat = 0 case.